### PR TITLE
 Add a generic Sobol' plot function 

### PR DESCRIPTION
**Pull request by jschueller**
Add SobolIndicesAlgorithm::DrawSobolIndices method to plot external Sobol indices.
We use it to fix chaos examples as mentioned in http://trac.openturns.org/ticket/958.

### CODE REVIEW

File: lib/src/Uncertainty/Algorithm/Sensitivity/SobolIndicesAlgorithmImplementation.cxx [hunk 1]
```diff
@@ -594,54 +594,73 @@ Sample SobolIndicesAlgorithmImplementation::computeIndices(const Sample & ,
   throw new NotYetImplementedException(HERE);
 }
 
-/** Method that draw (plot) the sensitivity graph */
-Graph SobolIndicesAlgorithmImplementation::draw() const
+
+Graph SobolIndicesAlgorithmImplementation::DrawSobolIndices(const Description & inputDescription,
+                                                            const Point & firstOrderIndices,
+                                                            const Point & totalOrderIndices)
 {
```
Comment (sofianehaddad) [inline lib/src/Uncertainty/Algorithm/Sensitivity/SobolIndicesAlgorithmImplementation.cxx]: we can also add a title argument

File: lib/src/Uncertainty/Algorithm/Sensitivity/SobolIndicesAlgorithmImplementation.cxx [hunk 2]
```diff
@@ -594,54 +594,73 @@ Sample SobolIndicesAlgorithmImplementation::computeIndices(const Sample & ,
...
+  Graph graph(DrawSobolIndices(inputDesign_.getDescription(), getAggregatedFirstOrderIndices(), getAggregatedTotalOrderIndices()));
+  if (outputDesign_.getDimension() > 1)
+    graph.setTitle(OSS() << "Aggregated Sobol' indices - " << getClassName());
+  else
+    graph.setTitle(OSS() << "Sobol' indices - " << getClassName());
+  const UnsignedInteger dimension = mergedFirstOrderIndices_.getDimension();
+
+  // Draw confidence intervals
   if (bootstrapSize_ > 0 && confidenceLevel_ > 0.0)
   {
```
Comment (sofianehaddad) [inline lib/src/Uncertainty/Algorithm/Sensitivity/SobolIndicesAlgorithmImplementation.cxx]: We might check if we use asymptotic estimation ? in that case, no need to have bootstrapSize_ > 0

File: lib/src/Uncertainty/Algorithm/Sensitivity/SobolIndicesAlgorithmImplementation.cxx (hunk 1 above)
Comment (jschueller) [reply lib/src/Uncertainty/Algorithm/Sensitivity/SobolIndicesAlgorithmImplementation.cxx]: It's easily set through the graph.

File: lib/src/Uncertainty/Algorithm/Sensitivity/SobolIndicesAlgorithmImplementation.cxx (hunk 2 above)
Comment (jschueller) [reply lib/src/Uncertainty/Algorithm/Sensitivity/SobolIndicesAlgorithmImplementation.cxx]: Indeed, fixed

**Pull request merged**